operators/labInstance-operator/k8s/operator/configmap.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: operator-config
+  namespace: lab-operator
+data:
+  WebdavSecretName: nextcloud-credentials
```

Comment:
I would change WebdavSecretName to webdavSecretName (first letter lower case), in order to uniform this name with the others and avoid future problems. The same modification should be performed in the deployment.